astro/src/content/docs/get-started/core-concepts/roles.mdx

```diff
@@ -121,7 +121,7 @@ _FusionAuth application roles_
 | `user_action_deleter`      | `631ecd9d-8d40-4c13-8277-80cedb8236f0` | Can delete user actions.                                                                                                                                                       |
 | `user_action_manager`      | `631ecd9d-8d40-4c13-8277-80cedb8236f1` | Can add and edit user actions.                                                                                                                                                 |
 | `user_deleter`             | `631ecd9d-8d40-4c13-8277-80cedb8236f2` | Can delete users.                                                                                                                                                              |
-| `user_manager`             | `631ecd9d-8d40-4c13-8277-80cedb8236f3` | Can add and edit users.<br/><br/>Please note that because this role can fully manage users, it is similar to `admin`. The `user_support_manager` is recommended in most cases. |
+| `user_manager`             | `631ecd9d-8d40-4c13-8277-80cedb8236f3` | Can add and edit users.<br/><br/>Please note that because this role can fully manage users, it is similar to `admin`. The `user_support_manager` is recommended in most cases. {/* eslint-disable-line */} |
```

Comment:
I could argue that we are using spacing to convey semantic information on this line instead of a proper tag, probably an `aside` (not our "Aside" component which does not wrap an `aside`). However, I think this happens all over the place currently so it is probably not worth fussing with the spacing at the moment.